python/pyspark/logger/tests/connect/__init__.py

```diff
@@ -0,0 +1,16 @@
+#
```

Comment:
this should also be added into `setup.py` in connect so the tests can run as a pure python package